doc/classes/VisualScriptEmitSignal.xml

```diff
@@ -5,6 +5,10 @@
 	</brief_description>
 	<description>
 		A Visual Script node which emits a specified signal when it is executed.
```

Comment:
You didn't edit this one. Following the rest of the PR: `Emits a specified signal when it is executed.`